src/bundle-configuration/zookeeper-prod/runtime-config.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+export ZK_DATA_DIR=/var/lib/zookeeper
```

Comment:
For the cassandra production bundle, we are using the directory `/var/lib/cassandra-v3` where cassandra-v3 consists of `$BUNDLE_NAME-v$BUNDLE_COMPATIBILITY_VERSION`.

Should we do the same for zookeeper? This is helpful in case you want to run several Zookeeper clusters at the same time.

Note that we are not using the `$BUNDLE_NAME-v$BUNDLE_COMPATIBILITY_VERSION` approach in conductr-elasticsearch either. Generally, I think we should be consistent here. Not namespacing the directory with the compatibility version would also simpify the cassandra prod bundle configuration because then we could put the directory into the `cassandra.yml` file.

@huntc @fsat WDYT?